.npmignore

```diff
@@ -15,3 +15,4 @@
 !index.js
 !init.js
 !package.json
+!vendor/*
```

Comment:
This should be at the end of the folders list (list is alphabetical but with folders at the top) and also `vendor/**/*` as I had issues in the past where subfolders were not getting included otherwise.